ada_url/ada_adapter.py

```diff
@@ -57,6 +53,35 @@ class HostType(IntEnum):
     IPV6 = 2
 
 
+class SchemeType(IntEnum):
+    """
+    Enum for URL scheme types:
```

Comment:
If you are interested, we can add a link to here to the actual URL standard, explaining what not special means in the context of URL: https://url.spec.whatwg.org/#url-miscellaneous